### PR TITLE
on|off is not supported in Boolean Filter anymore

### PR DESCRIPTION
**Pull request by antograssiot**
Ref https://github.com/api-platform/core/pull/964

### CODE REVIEW

**Comment by antograssiot** [comment]
I'made not sure if master is the good branch or if it should target 2.0...

File: core/filters.md [hunk 1]
```diff
@@ -152,7 +152,7 @@ constant directly.
 
 The boolean filter allows you to search on boolean fields and values.
 
-Syntax: `?property=[on|off|true|false|0|1]`
+Syntax: `?property=[true|false|0|1]`
```
Comment (teohhanhui) [inline core/filters.md]: true|false|1|0 to be consistent.

**Comment by antograssiot** [comment]
is there anything missing here that I can do/fix to get this documentation error fixed/merged ?

**Comment by teohhanhui** [comment]
Thanks @antograssiot!

**Pull request merged**